install/rpi_install_kalliope.sh

```diff
@@ -30,7 +30,7 @@ install_default_packages(){
     flac libffi-dev libffi-dev libssl-dev portaudio19-dev build-essential \
     libssl-dev libffi-dev sox libatlas3-base mplayer libyaml-dev libpython3-dev libjpeg-dev
     sudo apt-get install -y libportaudio0 libportaudio2 libportaudiocpp0  \
-    apt-transport-https
+    apt-transport-https pulseaudio
```

Comment:
the pulseaudio service should be added to the install as well, isn't it? And it needs to be added idempotently (check if not already there, then added and started)

Reply:
That would mean, we have to add a note to the section Raspberry Pi configuration in the docs, that if the script is used to install kalliope, the user can skip this step.

Reply:
My point is that user will expect to have kalliope working and testable with `kalliope start` righ after the installation.
If pulse is not started,it will not be the case.
What do you think?

Reply:
Yes you are right, and I'm currently working on the script, should be ready in few minutes